ui/job-view/headerbars/ReposMenu.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { getAllUrlParams } from '../../helpers/location';
+import { uiJobsUrlBase } from '../../helpers/url';
+
+export default class ReposMenu extends React.Component {
+  getRepoUrl(newRepoName) {
+    const params = getAllUrlParams();
+
+    params.delete('selectedJob');
+    params.set('repo', newRepoName);
+    return `${uiJobsUrlBase}?${params.toString()}`;
+  }
+
+  render() {
+    const { groupedRepos } = this.props;
+
+    return (
+      <span>
+        <span className="dropdown">
+          <button
+            id="repoLabel"
+            title="Watch a repo"
+            data-toggle="dropdown"
+            className="btn btn-view-nav nav-menu-btn dropdown-toggle"
+          >Repos</button>
+          <span
+            id="repo-dropdown"
+            className="dropdown-menu nav-dropdown-menu-right container"
+          >
+            <ul
+              className="checkbox-dropdown-menu row"
```

Comment:
The checkboxes are missing compared to production; is this intentional?
If so, no problem - just wanted to check (excuse the pun 😆).